tutorials/sinon/mock.md

```diff
@@ -0,0 +1,13 @@
+Mocks allow you to create a fake function that passes or fails depending on your needs.
+You can ensure it was called with certain arguments, or check how many times it was called.
+To complete the test, you must use the `verify()` function that checks if the mock did what you indicated.
+If not, it will throw an error.
+
+```javascript
+const sinon = require('sinon');
+const obj = { method: function() {console.log('Hello World')}};
+const mock = sinon.mock(obj);
+mock.expects("method").once();
+obj.method(); // 'Hello World' will not print
+mock.verify();
```

Comment:
Let's add an extra paragraph about `verify()`. Couple of things to clarify:

1) Does `verify()` throw an error if an expectation fails, or does it return true/false?
2) You need to define all expectations before calling `verify()`

Also, let's add another paragraph on `mock.restore()`